storage/src/backend/noop.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2020-2021 Alibaba Cloud. All rights reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//! Storage backend driver to access blobs on local filesystems.
```

Comment:
Storage backend driver to directly access cached blob data on local filesystems.